pyproject.toml

```diff
@@ -35,12 +34,12 @@ packages = [
 ]
 
 [tool.poetry.dependencies]  # https://poetry.eustace.io/docs/versions
-python = ">=3.6.4,<3.9"
+python = ">=3.7,<3.9"
 numpy = "^1.16"
 scipy = "^1.3"
 toml = "^0.10.0"
 pyquil = "^2.9"
-qctrl-open-controls = "^4.0.0"
+qctrl-open-controls = "^8.5.1"
```

Comment:
Maybe 9.0.0 (or is there any issue with the latest version)?
BTW, the bot doesn't seem to have updated the `setup.py` file in open control :(